### ISSUE TITLE
tasklite

### ISSUE THREAD

**Issue by xerus2000** [lyz-code/pydo#72]
Is the programming language really the only hurdle? Learning new programming languages can be a quite rewarding and interesting endeavor, especially if it follows a different paradigm.

And if there is no fundamental design incompatibility, how about using a compatible database format? The todo-space is already so fragmented...

**Comment by lyz-code** [comment]
I understand that learning new programming languages is interesting from several points of view and I won't lie to you, the biggest reason is driven by the sunk cost fallacy bias. Over the years I've built the knowledge and infrastructure to automate the creation python projects with (at least by my standards) high quality. Although I know I'm in the wrong for the long run, right now I'm not ready to spend that effort achieving that level in another language.

Regarding the database schema. I started with tasklite's one, but throughout the development of the program I needed additional fields while others weren't useful for our purposes, so I decided to drop that requirement. 

That won't be an inconvenient though, as tasklite supports [importing from taskwarrior and yaml](https://tasklite.org/migration.html) and we already support the [exporting as json](https://lyz-code.github.io/pydo/advanced_usage/export/), so you wouldn't have any problem moving from one to the other.

That being said, right now tasklite seems to be a more stable project, so until we reach 0.3.0 I won't recommend you to start using pydo unless you want to get involved in the development (which would be awesome :) ). Most of the project developers are currently stuck with exams so the development has currently slowed down a bit.
